.dockerignore

```diff
@@ -0,0 +1,9 @@
+.git
+node_modules
+.gitignore
+*.md
+Dockerfile
+*.sh
+deploy
+example
+*.yml
```

Comment:
Shouldn't the docker ignore also ignores /tests?
https://github.com/CoderDojo/cp-zen-platform/commit/e26f71f8f7e149318cdc9b2a404a9eb2e1c52c76#diff-f7c5b4068637e2def526f9bbc7200c4eR7